ui/src/app/pages/analysis/runtime-panel.tsx

```diff
@@ -1487,7 +1487,8 @@ const RuntimePanel = fp.flow(
                           runtimeCtx = {runtimeCtx}
                   />
               </FlexRow>
-              {currentRuntime && currentRuntime.errors && <ErrorMessage iconPosition={'top'} iconSize={16}>
+              {currentRuntime && currentRuntime.errors && currentRuntime.errors.length > 0
```

Comment:
Thanks, I was checking the other touch pointer. Double-check me but I think this other line in help-sidebar should be fine since we're checking the error state explicitly (and we'd probably want to show a modal, even if there were 0 errors returned):

https://github.com/all-of-us/workbench/blob/master/ui/src/app/components/help-sidebar.tsx#L436

Reply:
Checked testing manually, this is fine.